releases/release-1.26/README.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Kubernetes 1.26 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.26
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.26/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.26/release-team.md)
+* [Meeting Minutes](http://bit.ly/k8s126-releasemtg) (members of [release-team@] receive meeting invites)
+* [v1.26 Release Calendar][k8s126-calendar]
+* Contact: [#sig-release] on slack, [release-team](mailto://release-team@kubernetes.io) on e-mail
+* [Internal Contact Info] (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](https://bit.ly/k8s126-enhancements)
+* [Feature blog Tracking Sheet](http://bit.ly/k8s126-feature-blog)
+* [Bug Triage Tracking Sheet](https://bit.ly/k8s126-bug-triage-tracking)
+* [CI Signal Project Board & Issue Tracking](https://github.com/orgs/kubernetes/projects/68)
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.26 milestone](https://github.com/kubernetes/kubernetes/milestone/59)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## Summary
+
+The 1.26 release cycle is as follows:
+
+- **Monday 5th September 2022**: Week 1 — Release cycle begins
+- **Thursday 29th September 2022**: Week 4 — [Production Readiness Soft Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
+- **01:00 UTC Friday 7th October 2022** / **18:00 PDT Thursday 6th October 2022** : Week 5 — [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Wednesday 27th July 2022**: Week 11 — [Release Retrospective][Retrospective Document] part 1
+- **01:00 UTC Wednesday 23rd November 2022** / **02:00 BST Wednesday 23rd November 2022** / **18:00 PDT Tuesday 22nd November 2022**: Week 12 — [Code Freeze](../release_phases.md#code-freeze)
```

Comment:
a *lot* of people will be out of office this week for US holidays. I expected freeze earlier in november (like 11/8) to avoid holidays and give time for release candidates to get exercised and still release earlier in december